code/boot.js

```diff
@@ -141,7 +141,9 @@ function createDefaultBaseMapLayers() {
             stylers: [{visibility:"on"}, {hue:"#005eff"}, {invert_lightness:true}] },
           { featureType:"poi", stylers:[{visibility:"off"}]},
           { featureType:"transit", elementType:"all", stylers:[{visibility:"off"}] }
-        ]});
+        ],
+      needFixChinaOffset: true // managed by fix-china-map-offset plugin
+    });
   baseLayers['Google Roads'] = L.gridLayer.googleMutant({type:'roadmap', maxZoom: 21});
```

Comment:
Here and below, too need add
`needFixChinaOffset: true`

Reply:
So, we can add this option here, only for that layers that need it (excl. `hybrid` and `satellite`).
Otherwise we can leave `boot.js` untouched and patch layers options in plugin setup instead.
All layers are accessible via `layerChooser._layers`, and it's easy to patch only `_mutant` layers.

I'd prefer 2nd way.